src/linear_algebra/finite_dimensional.lean

```diff
@@ -1022,6 +1026,25 @@ lemma set_is_basis_of_linear_independent_of_card_eq_findim
   is_basis K (coe : s → V) :=
 is_basis_of_linear_independent_of_card_eq_findim lin_ind (trans s.to_finset_card.symm card_eq)
 
+lemma findim_one_implies_equal {F E : Type*} [field F] [field E] [algebra F E]
+  (E_dim : findim F E = 1) : (⊤ : subalgebra F E) = ⊥ :=
```

Comment:
```suggestion
lemma bot_eq_top_of_findim_eq_one {F E : Type*} [field F] [field E] [algebra F E]
  (E_dim : findim F E = 1) : (⊥ : subalgebra F E) = ⊤ :=
```

Reply:
I ended up naming this `subalgebra.bot_eq_top_of_findim_eq_one`. Also I ended up changing the proof and changed the lemma `findim_top` (which says that the `findim` of a vector space `V` over `K` is equal to the `findim` of `(⊤ : submodule K V)`) to not require that `V` is finite dimensional over `K`. Let me know if there are any problems.